airflow/jobs/backfill_job.py

```diff
@@ -921,6 +929,7 @@ def _execute(self, session: Session = NEW_SESSION) -> None:
             executor.end()
 
         self.log.info("Backfill done for DAG %s. Exiting.", self.dag)
+        return None
```

Comment:
(related nit: I'd rather we don't start the precedent of having these returns at the end of functions)

Reply:
No problem.